README.md

```diff
@@ -85,6 +85,13 @@ building and testing a bit easier. The following actions are available through t
     Generate the project code documentation and make it viewable
     locally.
 
+To run locally:
+    1. In api.yml at x-runtime->stats->output (around line 9) change “DATADOG” to “NULL”
+    2. In api.yml at paths->/v1/cloud/change->post->x-transportd->enabled (around line 78) comment out: - “metrics”
+    3. In api.yml at paths->/v1/cloud/ip/{ipAddress}->get->x-transportd->enabled (around line 127) comment out: - “metrics”
```

Comment:
I think you'll need to comment it out on the /ops paths too. Or just get rid of it entirely since this isn't the configuration used in production